neutrinordp/xrdp-neutrinordp.c

```diff
@@ -132,7 +132,7 @@ set_keyboard_overrides(struct mod *mod)
             /* When the keyboard type is 0x07 (Japanese keyboard) and the lower 16
                bits are a value of 0x0411(Japanese KeyLayout), the kbd_layout is set
                to 0x00000411 no matter what the upper 16 bits are. */
-            if ((settings->kbd_layout & 0x0000FFFF) == 0x00000411)
+            if ((mod->client_info.keylayout & 0x0000FFFF) == 0x00000411)
```

Comment:
I have changed the structure member variables to be checked, to make the intention clearer.